### ISSUE TITLE
SpaceAroundOperator doesn't autofix on multiline concatenated strings

### ISSUE THREAD

**Issue by mwallace72** [nvuillam/npm-groovy-lint#187]
npm-groovy-lint@9.0.0

Consider string 

```
sh 'curl --silent --show-error --fail '+
            "-u $u:$p "+
            "${artifactoryRepo}auth/* >> .npmrc"
```

Running with `--fix` will sort out the first line `+` not having a space, but fails to auto-format the following one. 
It does flag it in the output of the run.
